src/cmd/services/m3coordinator/handler/prometheus/native/read.go

```diff
@@ -38,7 +38,7 @@ import (
 
 const (
 	// PromReadURL is the url for native prom read handler
```

Comment:
Being a n00b, should a comment explain a bit more the difference between the native and remote endpoints?

Reply:
I'll let the author @benraskin92 update this one 😃